mgmtsystem_nonconformity_mrp/views/mgmtsystem_nonconformity_views.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!-- Copyright 2019 Marcelo Frare (Associazione PNLUG - Gruppo Odoo <http://odoo.pnlug.it>)
+     Copyright 2019 Stefano Consolaro (Associazione PNLUG - Gruppo Odoo <http://odoo.pnlug.it>)
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl) -->
+     
+<odoo>
+
+    <!-- inherit NC form view 
+            adding workcenter -->
+    <record id="view_mgmtsystem_nonconformity_form" model="ir.ui.view">
+        <field name="name">view.mgmtsystem.nonconformity.department</field>
+        <field name="model">mgmtsystem.nonconformity</field>
+        <field name="inherit_id" ref="mgmtsystem_nonconformity.view_mgmtsystem_nonconformity_form"/>
+        <field name="arch" type="xml">
+            <field name="create_date" position="after">
+                <field name="workcenter_id" groups="mrp.group_mrp_routings" />
```

Comment:
```suggestion
                <field name="workcenter_id"/>
```

Reply:
I think that's correct

Reply:
`groups="mrp.group_mrp_routings"` is a condition to show the workcenter only if `Work Orders & Quality` is checked on the Manufacturing settings.

Reply:
ok thanks @mymage for the clarification